src/pages/JobLogs.tsx

```diff
@@ -32,22 +63,44 @@ export const JobLogs = () => {
       ) : (
         <ContentWrapper>
           <TaskMetadata>
-            <H2>Build {buildId}</H2>
+            <H3>
+              {builder} – {buildNum}
+            </H3>
+            {taskId && (
+              <Body>
+                Task:{" "}
+                <StyledRouterLink
+                  to={getTaskRoute(taskId, { execution: taskExecution })}
+                  data-cy="task-link"
+                >
+                  {taskId}
+                </StyledRouterLink>
+              </Body>
+            )}
           </TaskMetadata>
 
           <CompleteLogsLink>
-            <Button data-cy="complete-test-logs-link" target="_blank">
+            <Button
+              href={getParsleyBuildLogURL(buildId)}
+              data-cy="complete-test-logs-link"
+              target="_blank"
+            >
               Complete logs for all tests
             </Button>
           </CompleteLogsLink>
+          {tests.length ? (
+            <JobLogsTable buildId={buildId} tests={tests} />
+          ) : (
+            <TablePlaceholder message="No test results found." />
```

Comment:
Sadly I do not think the task duration table works in this way 😢 LeafyGreen rejects non-`Row` children within its tables. But I moved the placeholder into `JobLogsTable`.

Reply:
The `TablePlaceholder` for the task duration table is actually outside of the Leafygreen `Table`! It just exists in a wrapper around the table as a workaround. So that's why you can continue seeing the headers:

<img width="1273" alt="Screen Shot 2023-02-15 at 5 24 27 PM" src="https://user-images.githubusercontent.com/47064971/219197521-a82da5d5-be53-4ab7-a336-c695b2f4cde3.png">

Reply:
ahhh I see, the difference is that there's no ternary to render the table _or_ the placeholder. Makes sense, ty!